### PR TITLE
Feat(eos_cli_config_gen): Add support for AAA accounting system commands

### LINKED ISSUE #1101: AAA accounting system commands
**Issue Type**

- [ ] Template enhancement
- [x] Role enhancement
- [ ] Documentation enhancement (avd.sh)

**Is your feature request related to a problem? Please describe.**

AAA accounting system commands should be configurable.

**Describe the solution you'd like**

aaa accounting system default structure needs to be defined.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by xaviramon**
## Change Summary

EOS CLI config gen templates update to support AAA accounting system default commands.
Includes molecule updated tests.

Can generate conflicts when merging with PR#1132. In that case, command "aaa accounting system default" should be rendered immediately under "aaa accounting exec default" command
## Related Issue(s)

Fixes #1101 

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
New structure defined for aaa accounting system default commands

## How to test
Create structure or validate with molecule

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by titom73 — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-accounting.j2
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-accounting.j2]: On `exec` above we print `-` and not `Default`. We should align to one or the other.

**Review by ClausHolbechArista — COMMENTED** [review]
This is related to PR #1132 which is still pending. In 1132 we change the layout of documentation. If this is in a hurry to get in before 1132, please update 1132 after this one has merged.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-accounting.j2
Comment (xaviramon) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-accounting.j2]: Different options are available for each command, under 4.25.2:
```
POD1_SPINE01(config)#aaa accounting exec ?
  console  Console accounting method list
  default  The default accounting list

POD1_SPINE01(config)#aaa accounting system ?
  default  The default accounting list
```
This can wait until PR#1132 is merged.

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-accounting.j2]: 1132 is merged now, so this one needs to be adjusted to the new documentation layout.

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**
